dev/fattest.simplicity/src/componenttest/topology/utils/MvnUtils.java

```diff
@@ -979,14 +980,64 @@ public static int runCmd(String[] cmd, File workingDirectory, File outputFile) t
         pb.redirectErrorStream(true);
 
         Log.info(c, "runCmd", "Running command " + Arrays.asList(cmd));
+
+        int hardTimeout = Integer.parseInt(System.getProperty("fat.timeout"));
```

Comment:
is `fat.timeout` always guaranteed to be set? May want to default to some high number if unset